internal/cypress/sauce/runner.go

```diff
@@ -3,20 +3,26 @@ package sauce
 import (
 	"context"
 	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
 	"github.com/rs/zerolog/log"
+
 	"github.com/saucelabs/saucectl/cli/credentials"
 	"github.com/saucelabs/saucectl/cli/progress"
 	"github.com/saucelabs/saucectl/internal/archive/zip"
 	"github.com/saucelabs/saucectl/internal/cypress"
 	"github.com/saucelabs/saucectl/internal/job"
 	"github.com/saucelabs/saucectl/internal/jsonio"
+	"github.com/saucelabs/saucectl/internal/resto"
 	"github.com/saucelabs/saucectl/internal/storage"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"time"
 )
 
+// SauceLabsURL represents sauce labs app URL.
+var SauceLabsURL = "https://app.saucelabs.com"
```

Comment:
@valentyn88 we need to be able to handle regions, check how are we hadlong it with testcomposer
https://github.com/saucelabs/saucectl/blob/cac248e103f8454d2aa929a9d3f27d69ec0ac2d9/internal/testcomposer/testcomposer.go#L52

Reply:
We can set different values during build time `go build -ldflags="-X 'sauce.sauceLabsURL=new_value'"`. Can it solve our problem?

Reply:
I think that won't be ok since the same binary should be able to reach both us-west-1 (app.saucelabs.com) and eu-central-1 (app.eu-central-1.saucelabs.com)

Reply:
Can I set this URL based on sauce.region from config,yml ?

Reply:
Not only the region, tld should be also considered. There are 3 urls I actually met.
us-west-1 (app.saucelabs.com) 
eu-central-1 (app.eu-central-1.saucelabs.com) 
staging (app.staging.saucelabs.net)

Reply:
That is handled already https://github.com/saucelabs/saucectl/pull/127/files

Reply:
Great!